tests/test_build.py

```diff
@@ -17,24 +17,28 @@ def get_conda_list_dict():
     return conda_list_dict
 
 
-@pytest.mark.skip
 def get_meta_packages_dict():
     """Get packages and versions specified on meta.yaml as a dictionary"""
-    with open("continuous_integration/recipe/meta.yaml", "r") as f:
-        meta = yaml.safe_load(f)
+    env = Environment(
+        loader=FileSystemLoader(
+            pathlib.Path(__file__).parent.parent / "continuous_integration/recipe"
+        ),
+        autoescape=select_autoescape(),
+    )
+    template = env.get_template("meta.yaml")
+    meta = yaml.safe_load(template.render())
```

Comment:
It looks like we recently switched to using jinja templating in `meta.yaml` (my guess is this is for consistency with conda-forge) which resulted in failing tests because jinja templating can lead to invalid YAML formatting. We now render the jinja template before attempting to do any YAML parsing to avoid this error